docs/technical-docs/issues-workflow/psychopy.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: default
+title: Custom Workflows
+parent: General Issues Workflow
+---
+
+# Custom Workflows
+
+## Overview
+
+Custom Issues workflow for specific projects in the lab where the issues are used for more than just
```

Comment:
This reads like a long title? Is the word "are" before "for" missing?

Maybe add an example or two?